lib/wicked_pdf/wicked_pdf_helper/assets.rb

```diff
@@ -27,6 +29,32 @@ def wicked_pdf_stylesheet_link_tag(*sources)
         end.html_safe
       end
 
+      def wicked_pdf_stylesheet_pack_tag(*sources)
+        return unless defined?(Webpacker)
+        if running_in_development?
+          stylesheet_pack_tag(*sources)
+        else
+          css_text = sources.collect do |source|
+            source = WickedPdfHelper.add_extension(source, 'css')
+            wicked_pdf_stylesheet_link_tag(webpacker_source_url(source))
```

Comment:
Just dropping in, so these questions are probably more ignorance:

1 • The JS pack tag uses `#read_asset` to resolve the url/path, why does that not apply here?
2 • In non dev-server environments, should be both the stylesheet and js tags use a path helper instead of a url helper?

Reply:
RE #2: It looks like `#webpacker_source_url` does some of this work.